pkg/cli/config.go

```diff
@@ -27,22 +27,22 @@ import (
 )
 
 type GCPHostConfig struct {
-	MachineType    string
-	MinCPUPlatform string
+	MachineType    string `json:"machine_type,omitempty"`
+	MinCPUPlatform string `json:"min_cpu_platform,omitempty"`
 }
 
 type HostConfig struct {
-	GCP GCPHostConfig
+	GCP GCPHostConfig `json:"gcp,omitempty"`
 }
 
 type Config struct {
-	ServiceURL                string
-	Zone                      string
-	HTTPProxy                 string
-	ConnectionControlDir      string
-	KeepLogFilesDays          int
-	BuildAPICredentialsSource string
-	Host                      HostConfig
+	ServiceURL                string     `json:"service_url,omitempty"`
```

Comment:
Doesn't this break parsing the config files?
The config file has `ServiceURL` instead of `service_url`

Reply:
it does not break, the toml parsing do not uses the json annotations.